changes/4199.dependencies

```diff
@@ -0,0 +1 @@
+Updated `cryptography` to `~41.0.3`.
```

Comment:
```suggestion
Updated `cryptography` to `~41.0.3`. As this is not a direct dependency of Nautobot, it will not auto-update when upgrading. Please be sure to upgrade your local environment.
```